packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -190,8 +195,41 @@ class HTTPCommunicationQueue {
     await this.#abortMultipartUpload(file, awaitedResult)
   }
 
+  async #nonMultipartUpload (file, chunk, signal) {
+    const filename = file.meta.name
+    const { type } = file.meta
+    const metadata = undefined // TODO: add `allowedMetaFields` option to S3 multipart?
```

Comment:
should we pull out the `const metadata = ...` code from createMultipartUpload into a common function and use that?